app/assets/javascripts/client/home/StartAssessmentCtrl.js

```diff
@@ -0,0 +1,97 @@
+(function() {
+  'use strict';
+  angular.module('PDRClient')
+      .controller('StartAssessmentCtrl', StartAssessmentCtrl);
+
+  StartAssessmentCtrl.$inject = [
+    '$scope',
+    '$timeout',
+    '$location',
+    '$modal',
+    'Assessment',
+    'SessionService'
+  ];
+
+  function StartAssessmentCtrl($scope, $timeout, $location, $modal, Assessment, SessionService) {
+    var vm = this;
+
+    vm.user = SessionService.getCurrentUser();
+    vm.district = vm.user.districts[0];
+
+    vm.alerts = [];
+    vm.assessment = {};
+
+    vm.hideModal = function() {
+      $('#startAssessment').modal('hide');
+    };
+
+    vm.noDistrict = function() {
+      return _.isEmpty(vm.user.district_ids);
+    };
+
+    vm.redirectToAssessment = function(assessment) {
+      vm.hideModal();
+      $location.path('assessments/' + assessment.id + '/assign');
+    };
+
+    vm.create = function(assessment) {
+      assessment.due_date = moment($("#due-date").val(), 'MM-DD-YYYY').toISOString();
```

Comment:
I'd love to unify this at some point as well since we should only have one way to do this.  But it's likely not in the scope of this task.  This looks like a hold over from the previous controller that existed.